coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
+    """
+    Escapes the given string with [c] pattern
+
+    :param string: String that is to be escaped with [ ]
+    :return:       Escaped string in which all the special glob characters
+                   such as ()[]!? are escaped.
+    """
+    output = ""
+    special_chars = "()[]|?*"
+    for char in string:
+        if char in special_chars:
+            output += "[" + char + "]"
+        else:
+            output += char
+    return output
+
```

Comment:
Isn't the current version simpler? I agree that your solution is shorter (and I'd personally prefer it on personal projects), but I feel the current version is more readable and less prone to errors due to carelessness in the future (and more explicit in my opinion). But if this is how it is done coala wide, I'm perfectly okay with changing it.